engine/olcPixelGameEngine.h

```diff
@@ -854,8 +854,8 @@ namespace olc
 
 	Pixel Sprite::Sample(float x, float y)
 	{
-		int32_t sx = (int32_t)(x * (float)width);
-		int32_t sy = (int32_t)(y * (float)height);
+		uint32_t sx = (uint32_t)(x * (float)width);
+		uint32_t sy = (uint32_t)(y * (float)height);
 		return GetPixel(sx, sy);
```

Comment:
We're referencing a not existing object within [`return Pixel(0, 0, 0, 0);`](https://github.com/philsupertramp/wirdnix/blob/a87884755142f1aeb53d5a10fa3abfa2a1ae1297/engine/olcPixelGameEngine.h#L834)

Reply:
change ` return Pixel(0, 0, 0, 0)` to `return olc::BLANK;`

Reply:
is the change to explicitly unsigned the killer for you? or was it something else that changed for better performance on your end?